config/locales/ar/browse.yml

```diff
@@ -2,5 +2,5 @@
 ar:
   browse:
     all_categories: جميع الفئات
-    description: 
-    title: 
+    description:
```

Comment:
Minor nitpick, this looks unintentional. Would be good for this not to appear in the change history for this PR

Reply:
Thanks for pointing this out 👍